tests/js/common/aql/aql-geo-distance-range.js

```diff
@@ -54,14 +51,15 @@ function BTS_470() {
   
       let lat = 6.537;
       let long = 50.332;
-      for (let x = 0; x < 500; ++x) {
-        let points = [];
-        for (let y = 0; y < 500; ++y) {
+      let points = [];
+      for (let x = 0; x < 100; ++x) {
+        for (let y = 0; y < 100; ++y) {
```

Comment:
This is valid because other coords was out of bound box queried data or it doesn't changes because it was strictly interior